src/transformers/models/levit/convert_timm_to_pytorch.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team.
```

Comment:
You can rename this script to `convert_levit_timm_to_pytorch.py`

Reply:
Done

Reply:
Doesn't seem to be done